base/inference.jl

```diff
@@ -3276,44 +3276,6 @@ function replace_getfield!(ast, e::ANY, tupname, vals, sv, i0)
     end
 end
 
-function code_typed(f, types::ANY; optimize=true)
-    types = to_tuple_type(types)
-    code_typed(call, Tuple{isa(f,Type)?Type{f}:typeof(f), types.parameters...}, optimize=optimize)
```

Comment:
@vtjnash Somehow I always thought you what you meant is that the code in this PR is having problem getting the correct `call`.

So should I do the one-liner (ok maybe two since there's also `return_types`) and experimenting with moving these out of `Inference` in another PR?